src/scenic/simulators/webots/model.scenic

```diff
@@ -154,7 +264,7 @@ class Ground(WebotsObject):
                 heightField.insertMFFloat(-1, 0)
         # Set height values
         i = 0
-        for row in self.heights:
+        for row in [row[::-1] for row in self.heights]:
             for height in row:
```

Comment:
This unnecessarily duplicates the entire heightmap. If it is really necessary to reverse the rows here I would do:
```python
for row in self.heights:
    for height in reversed(row):
```
But it would be better to just create the heightmap in the desired order to begin with.